src/main/java/eu/interop/federationgateway/validator/DiagnosisKeyBatchValidator.java

```diff
@@ -15,24 +17,59 @@
   @Override
   public boolean isValid(EfgsProto.DiagnosisKeyBatch diagnosisKeyBatch, ConstraintValidatorContext context) {
 
+    /*
+     see
+       - https://github.com/google/exposure-notifications-server/blob/5a77e6982b16c7282c77dea39772491cc7b7dc8b/internal/publish/model/exposure_model.go#L338
+       - https://github.com/google/exposure-notifications-server/blob/5a77e6982b16c7282c77dea39772491cc7b7dc8b/tools/export-analyzer/main.go#L123
+     */
+
+    long minimumRollingStart = Instant
+      .now()
+      .truncatedTo(ChronoUnit.DAYS)
+      .minus(15, ChronoUnit.DAYS)
+      .getEpochSecond() / 600;
```

Comment:
recommend putting 600 as a named constant that describes the meaning

Reply:
Aggree with you. fixed.